prov/rxm/src/rxm_ep.c

```diff
@@ -1640,8 +1717,7 @@ void rxm_ep_progress_deferred_queue(struct rxm_ep *rxm_ep,
 				rxm_cq_write_error(
 					def_tx_entry->rxm_ep->util_ep.rx_cq,
 					def_tx_entry->rxm_ep->util_ep.rx_cntr,
-					def_tx_entry->rndv_read.rx_buf->
-						recv_entry->context, ret);
+					NULL, ret);
```

Comment:
This change looks like it should be separate.  Explain why replacing context with NULL is valid.